src/views/datasets/ri-buoy/summary.vue

```diff
@@ -3,15 +3,26 @@
     <template #available-data>
       This dataset spans from {{ store.minDate.slice(0, 4) }} to
       {{ store.maxDate.slice(0, 4) }} and was collected by the Narragansett Bay
-      Fixed Station Monitoring Network (NBFSMN) with RIDEM-OWR as the lead
-      agency. Agencies involved in collection and maintenance of the data are:
-      RIDEM-OWR, URI/GSO MERL, NBC, NBNERR, and MassDEP. The heatmap below
-      summarizes the number of observations collected for each month for
-      different variables. Use this heatmap to help you decide what data you
-      want to visualize or download. When you have an idea, go ahead and select
-      the buoys, variables and dates to explore. Or download the data in the
-      most appropriate format for your analyses! To begin, select a variable to
-      see what data is available.
+      Fixed Station Monitoring Network (<a
+        href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
+        >NBFSMN</a
+      >) with <a href="http://www.dem.ri.gov/programs/water/">RIDEM-OWR</a> as
```

Comment:
make all of these links `target="_blank" rel="noreferrer noopener"` so they open in a new tab (if we're feeling fancy, we could even pull that out into a mini-component, but might be overkill here)